CondFormats/GEMObjects/src/GEMeMap.cc

```diff
@@ -81,13 +81,16 @@ void GEMeMap::convertDummy(GEMROMapping& romap) {
   for (int re = -1; re <= 1; re = re + 2) {
     for (int st = GEMDetId::minStationId0; st <= GEMDetId::maxStationId; ++st) {
       int maxVFat = maxVFatGE11_;
+      int maxLayerId = GEMDetId::maxLayerId;
       if (st == 2)
```

Comment:
Please use these enums instead https://github.com/cms-sw/cmssw/blob/master/DataFormats/MuonDetId/interface/GEMSubDetId.h#L12 `GEMSubDetId::Station::GE21`

Reply:
The st equality test is updated in the new commit